ldif.go

```diff
@@ -174,7 +174,7 @@ func (l *LDIF) parseEntry(lines []string) (entry *Entry, err error) {
 	}
 
 	if !strings.HasPrefix(lines[0], "dn:") {
-		return nil, errors.New("Missing dn:")
+		return nil, errors.New("missing 'dn:'")
```

Comment:
Good catch on the capitalization; what about `missing 'dn':`? (colon outside the single quotes)